src/Uno.UI/UI/Xaml/FrameworkElement.wasm.cs

```diff
@@ -443,15 +443,5 @@ private protected override void UpdateDOMProperties()
 				base.UpdateDOMProperties();
 			}
 		}
-
-		public override string ToString()
-		{
-			if (FeatureConfiguration.UIElement.RenderToStringWithId && !Name.IsNullOrEmpty())
-			{
-				return $"{base.ToString()}\"{Name}\"";
-			}
```

Comment:
That part was performance relate, iirc. Did you move it somewhere else?

Reply:
Yes, moved to `FrameworkElement.crossruntime.cs`

The problem with the current approach is that if in your app you did something like this:

```csharp
public class MyCustomFE : FrameworkElement
{
    // This DOES NOT call FrameworkElement.ToString. It calls object.ToString!
    // This is because the reference API doesn't have FrameworkElement.ToString so the compiler translates this as a call to object.ToString because that is what the compiler finds in compile-time.
    public override string ToString() => base.ToString();
}
```